plugins/UsersManager/API.php

```diff
@@ -513,6 +514,22 @@ public function getUsersHavingSuperUserAccess()
     }
 
     /**
+     * Regenerate the token_auth associated with a user.
```

Comment:
> grab the authentication and force a re-authentication with the regenerated token

I'm not quite sure what you mean. It is a rare action so pretty much anything should be fine I'd say :)